tests/acceptance/features/apiFullTextSearch/search.feature

```diff
@@ -25,3 +25,23 @@ Feature: full text search
       | old              |
       | new              |
       | spaces           |
+
+
+  Scenario Outline: search restored files using content
+    Given using <dav-path-version> DAV path
+    And user "Alice" has uploaded file with content "hello world" to "file1.txt"
+    And user "Alice" has uploaded file with content "Namaste nepal" to "file2.txt"
```

Comment:
IMO, a descriptive file name convention would be better. Also, maintain uniformity throughout the scenarios.